Makefile

```diff
@@ -100,7 +100,11 @@ else
 	JSON=$(JSON) VERSION=$(VERSION) scripts/makejson
 endif
 
-clean:
+# Clean files produced by "wad-diff".
+wad-diff-clean:
+	rm -f wad-images/*.png
```

Comment:
Understood, but no reason to be intimated by changing what is here. [Be bold!](https://en.wikipedia.org/wiki/Wikipedia:Be_bold)